tests/base/tensor_accessors_02.cc

```diff
@@ -76,8 +76,10 @@ int main()
 
     // via std::array
 
+    // via std::array:
 #ifdef DEAL_II_WITH_CXX11
-    deallog << TensorAccessors::extract<5>(foo, std::array<unsigned int, 5>({2, 1, 0, 2, 1})) << std::endl;
```

Comment:
It's not quite the same as far as I understand it.
```
  classname{a,b,c}
```
first tries to match the arguments `a,b,c` to a constructor that takes three arguments (or more if some are defaulted, or to `...` arguments) and, I believe, only then tries to call the class with a `std::initializer_list` argument. If you write
```
  classname({a,b,c})
```
you always get the initializer list argument. I don't think it makes a difference here.